pkg/jwx/jwx.go

```diff
@@ -84,13 +84,32 @@ func DecodeAccessToken(accessToken string, e, n *string) (*jwt.Token, *jwt.MapCl
 	}
 
 	claims := &jwt.MapClaims{}
+
+	audienceToCheck := ""
+	if len(expectedAudience) != 0 {
+		audienceToCheck = expectedAudience[0]
```

Comment:
is there no need to check the other audiences?

Reply:
if the audience to check is provided it is checked using the functional option in line 97

Reply:
if it is not provided, it will be ignored using the functional option in line 112

Reply:
oh i misread your comment. 
There shouldn't be more than one audience to check if i'm not totally wrong